app/text_buffer.py

```diff
@@ -49,6 +49,9 @@ def __init__(self):
     self.parserTime = .0
     self.relativePath = ''
     self.redoChain = []
+    self.tempChain = []  # Used to store cursor view actions without trimming redoChain
```

Comment:
Does the tempChain every get more than one element in it?
i.e. would it make sense to make this a tempChange or something rather than a tempChain